flightcheck/appHook.js

```diff
@@ -50,13 +55,12 @@ class AppHook {
     this.information = Object.assign(this.information, obj)
   }
 
-  // TODO: flatten data object
-  // TODO: base64 decode all files automaticly?
   file (path) {
-    return Request
+    return request
     .get(`https://api.github.com/repos/${this.data.project.github.fullName}/contents/${path}?ref=${this.data.tag}`)
```

Comment:
are we still not checking the code out? Or is this in one of your newer branches?